vars/main.yml

```diff
@@ -1,4 +1,6 @@
 __timesync_chrony_version: "{{
-  ansible_facts.packages['chrony'][0].version|default('')|int }}"
+  (( ansible_facts.packages.chrony | default([]) )[0]
+    | default(dict()) )['version'] | default('0') }}"
 __timesync_ntp_version: "{{
-  ansible_facts.packages['ntp'][0].version|default('')|int }}"
+  (( ansible_facts.packages.ntp | default([]) )[0]
+    | default(dict()) )['version'] | default('0') }}"
```

Comment:
I think it can be shortened by using `d()` instead of `default()` and `{}` instead of `dict`. Still a bit ugly though and the logging solution may be better.